src/query/functions/src/scalars/geometry.rs

```diff
@@ -1502,109 +1508,112 @@ pub fn register(registry: &mut FunctionRegistry) {
         ),
     );
 
-    // registry.register_passthrough_nullable_2_arg::<GeometryType, Int32Type, GeometryType, _, _>(
```

Comment:
These is no used anymore?

Reply:
Yes, those code was commented out earlier because it could not solve the sqlite dependency problem. Now we have implemented the `proj` algorithm using `proj4rs` implemented by Rust. The previous code is no longer needed and can be deleted.